.github/workflows/bazel.yml

```diff
@@ -184,7 +186,7 @@ jobs:
             cd /workspaces/magma
             bazel test //... --test_output=errors --cache_test_results=no
       - name: Run `bazel run //:check_starlark_format`
-        uses: addnab/docker-run-action@v2
+        uses: addnab/docker-run-action@v3
```

Comment:
Should pull always be added here as well?
I assume it needs only to be added to the very first use of the image so it will get updated in the local repository - but for consistency I recommend to add it in every step where the image is used.

Reply:
No, as detailed in the PR description.

Reply:
Ah, I did not see that it was only in the first for each step - ok.